### DISCUSSION TITLE
@prisma/client is not in the npm registry, or you have no permission to fetch it.

### DISCUSSION THREAD

**Discussion by jake-davisOVO** [prisma/prisma#24799] [Q&A]
### Bug description

I'm working in a turborepo monorepo and when I'm on main and run `pnpm i`

I get the following error:

`@prisma/client is not in the npm registry, or you have no permission to fetch it.`

if I try to run `% npm install prisma@5.14.0 --save`

I get the following..

 npm install prisma@5.14.0 --save

npm ERR! code E404
npm ERR! 404 Not Found - GET https://npm.pkg.github.com/@prisma%2fengines - npm package "engines" does not exist under owner "prisma"
npm ERR! 404 
npm ERR! 404  '@prisma/engines@5.14.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

### Prisma information

N/A as this is an installation issue

### Environment & setup

- Operating system: MAC OS
- Database: PostgreSQL
- Node.js version: v18.18.0

### Prisma Version

```
N/a not installed
```

**Comment by ludralph** [comment]
Hi @jake-davisOVO 👋 

Can you try running the command `pnpm add @prisma/client`? Do let us know if you run into any errors.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
